docs/source/troubleshooting.rst

```diff
@@ -0,0 +1,211 @@
+
+Installation Issues
+-------------------
+
+**ImportError: No module named 'tacs'**
+
+*Cause*: TACS is not installed or not in the Python path.
+
+*Solutions*:
+
+- Ensure conda environment is activated: ``conda activate tacs``
+- Reinstall TACS: ``mamba install -c conda-forge -c smdogroup tacs``
+- For source installation, ensure Python interface is built: ``pip install -e .[all]``
+
+**ImportError: libtacs.so not found**
+
+*Cause*: TACS C++ library is not in the library path.
+
+*Solutions*:
+
+.. code-block:: bash
+
+   # Add TACS lib directory to LD_LIBRARY_PATH
+   export LD_LIBRARY_PATH=/path/to/tacs/lib:$LD_LIBRARY_PATH
+   
+   # Or add to your .bashrc for permanent fix
+   echo 'export LD_LIBRARY_PATH=/path/to/tacs/lib:$LD_LIBRARY_PATH' >> ~/.bashrc
+
+**MPI compilation errors**
+
+*Cause*: MPI compiler not found or misconfigured.
+
+*Solutions*:
+
+- Ensure MPI is installed: ``which mpicc``
+- Check MPI compiler in Makefile.in: ``CXX = mpicxx``
+- For Intel MPI: ``CXX = icpc -lmpi``
+
+Analysis Issues
+---------------
+
+**Analysis fails to converge**
+
+*Cause*: Poor conditioning, incorrect boundary conditions, or excessive loads.
+
+*Solutions*:
+
+- Check boundary conditions are properly applied
+- Verify loads are reasonable for the structure
+- Ensure material properties are correct
+- Try reducing load magnitude
+- Check for rigid body modes
+
+**Element callback errors**
+
+*Cause*: Mismatch between BDF element types and callback function.
+
+*Solutions*:
+
+- Verify element types in BDF file match callback function
+- Add error handling for unexpected element types:
+
+  .. code-block:: python
+
+     for descript in elemDescripts:
+         if descript == 'CQUAD4':
+             elem = elements.Quad4Shell(transform, con)
+         else:
+             raise ValueError(f"Unexpected element type: {descript}")
+
+**Memory errors with large models**
+
+*Cause*: Insufficient memory for analysis or postprocessing.
+
+*Solutions*:
+
+- Consider using more powerful hardware
+
+Postprocessing Issues
+--------------------
+
+**f5tovtk/f5totec not found**
+
+*Cause*: Postprocessing utilities not installed or not in PATH.
+
+*Solutions*:
+
+.. code-block:: bash
+
+   # Add to PATH
+   export PATH=/path/to/tacs/extern/f5tovtk:$PATH
+   export PATH=/path/to/tacs/extern/f5totec:$PATH
+   
+   # Or use full path
+   /path/to/tacs/extern/f5tovtk/f5tovtk solution.f5
+
+**Conversion fails with large files**
+
+*Cause*: Insufficient disk space or memory.
+
+*Solutions*:
+
+- Use essential output flags only
+- Convert files individually
+
+Performance Issues
+------------------
+
+**Slow analysis performance**
+
+*Cause*: Suboptimal settings or hardware limitations.
+
+*Solutions*:
+
+- Use appropriate number of MPI processes
+- Enable optimized BLAS/LAPACK (Intel MKL)
+- Use SuiteSparse for better sparse matrix performance
+- Check system resources: ``htop``
```

Comment:
These bullet points are not particularly useful unless they explain how to do these things, or link to somewhere that does. For example, how do I know what an "appropriate number of MPI processes" is? How do I enable TACS to use MKL? What does it mean to "Use SuiteSparse for better sparse matrix performance" and how do I do it? What should I be checking `htop` for?